plugindocs.rb

```diff
@@ -11,7 +11,7 @@
 
 class PluginDocs < Clamp::Command
   option "--output-path", "OUTPUT", "Path to the top-level of the logstash-docs path to write the output.", required: true
-  option "--master", :flag, "Fetch the plugin's docs from master instead of the version found in PLUGINS_JSON", :default => false
```

Comment:
Do occurrences in lines 35 and 120 need to change?

Reply:
indeed, good catch!